lib/types.nix

```diff
@@ -451,6 +451,8 @@ rec {
       nestedTypes.elemType = elemType;
     };
 
+    niceness = types.ints.between (-20) 19;
```

Comment:
Sorry for the nit, but I think this'd be better placed near the `int` type (right below where `ports` is defined).